[2024-05-19T19:27:11.406+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T19:27:11.412+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: disney_api_to_s3.save_to_s3 scheduled__2024-05-19T19:20:00+00:00 [queued]>
[2024-05-19T19:27:11.415+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: disney_api_to_s3.save_to_s3 scheduled__2024-05-19T19:20:00+00:00 [queued]>
[2024-05-19T19:27:11.416+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-19T19:27:11.420+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): save_to_s3> on 2024-05-19 19:20:00+00:00
[2024-05-19T19:27:11.423+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=616) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T19:27:11.424+0000] {standard_task_runner.py:63} INFO - Started process 618 to run task
[2024-05-19T19:27:11.425+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'disney_api_to_s3', 'save_to_s3', 'scheduled__2024-05-19T19:20:00+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/api_disney.py', '--cfg-path', '/tmp/tmpu6fw3uu8']
[2024-05-19T19:27:11.426+0000] {standard_task_runner.py:91} INFO - Job 108: Subtask save_to_s3
[2024-05-19T19:27:11.443+0000] {task_command.py:426} INFO - Running <TaskInstance: disney_api_to_s3.save_to_s3 scheduled__2024-05-19T19:20:00+00:00 [running]> on host 620090466bc8
[2024-05-19T19:27:11.479+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='disney_api_to_s3' AIRFLOW_CTX_TASK_ID='save_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-05-19T19:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-19T19:20:00+00:00'
[2024-05-19T19:27:11.482+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T19:27:11.495+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-05-19T19:27:11.506+0000] {base.py:84} INFO - Using connection ID 'aws_s3' for task execution.
[2024-05-19T19:27:11.507+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-05-19T19:27:14.059+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-19T19:27:14.061+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T19:27:14.081+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=disney_api_to_s3, task_id=save_to_s3, run_id=scheduled__2024-05-19T19:20:00+00:00, execution_date=20240519T192000, start_date=20240519T192711, end_date=20240519T192714
[2024-05-19T19:27:14.118+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-19T19:27:14.133+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-19T19:27:14.134+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
